### PR TITLE
store - remove collection updater, rewrite collection edit dialog

### PR DESCRIPTION
**Pull request by janicka1**
Feature #14103

Arvados-DCO-1.1-Signed-off-by: Janicki Artur <artur.janicki@contractors.roche.com>

### CODE REVIEW

**Comment by janicka1** [comment]
@burzomir No ;) Because this will be only temporary solution. We need to create and move all text and snackbar to new service.

File: src/views-components/dialog-create/dialog-collection-create.tsx [hunk 1]
```diff
@@ -3,113 +3,38 @@
...
+import { FileUpload } from '~/components/file-upload/file-upload';
+
+// interface DialogCollectionDataProps {
+//     open: boolean;
+//     handleSubmit: any;
+//     submitting: boolean;
+//     invalid: boolean;
+//     pristine: boolean;
+//     files: UploadFile[];
+// }
```
Comment (unodgs) [inline src/views-components/dialog-create/dialog-collection-create.tsx]: Why not removed?

File: src/views-components/form-fields/project-form-fields.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+import * as React from "react";
+import { Field, WrappedFieldProps } from "redux-form";
```
[the remark this answers is not in the document]
Comment (janicka1) [reply src/views-components/form-fields/project-form-fields.tsx]: Removed

File: src/views-components/dialog-create/dialog-collection-create.tsx (hunk 1 above)
Comment (janicka1) [reply src/views-components/dialog-create/dialog-collection-create.tsx]: Because, we created a task about improve files for collection. I left for this.

**Pull request merged**